docs/tag_discovery.md

```diff
@@ -0,0 +1,17 @@
+# Tags Discovery Behavior for Multibranch Pipeline Jobs
+
+## Overview
+As of version 4.1.0 of this plugin, we have added a new tag discovery behavior for Multibranch Pipeline jobs. 
+This behavior allows the job to discover tags in Bitbucket Data Center.
+
+:warning: This change will only discover tags. We currently do not support automatic tag builds.
```

Comment:
@jaguilar-atl might explain this better but I wonder if we need this comment? Other behaviours that do this exist, so I don't know if we specifically need to provide one.

Reply:
Was thinking it might be better to exclusively call it out, as we may immediately get bug reports for it without conveying it first.